english/marshmallow.md

```diff
@@ -35,6 +35,7 @@ January 2017 approaches. You have two presidential candidates to vote for.
 
 You don't like either of these candidates.
 So instead, you decide to [write a symphony](write-a-symphony/symphony.md)
+or [paint a picture](paint-a-picture.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.